src/@carbon-info/views/Ecosystem/Ecosystem.tsx

```diff
@@ -126,6 +128,12 @@ const Ecosystem: React.FC = () => {
     return filtered;
   }, [allDAppsCore, allDAppsEVM, dAppsFilter]);
 
+  const sortedDApps= React.useMemo(() => {
+    return filteredDApps.sort((v1: DAppsConfig, v2: DAppsConfig) => {
+      return v1.sortPriority - v2.sortPriority;
+    });
+  }, [allDAppsCore, allDAppsEVM, dAppsFilter]);
```

Comment:
Think the sort function can be moved into the `useMemo` on L115. Then there's no need for another `useMemo`